app/services/imo_api_service.rb

```diff
@@ -0,0 +1,71 @@
+class ImoApiService
+  IMO_USERNAME = ENV["IMO_USERNAME"]
+  IMO_PASSWORD = ENV["IMO_PASSWORD"]
+  BASE_URL = "https://sgp.imo.im/api/simple/"
+
+  class ImoApiService::HTTPError < HTTP::Error
+  end
+
+  attr_reader :phone_number, :recipient_name, :locale
+
+  def initialize(phone_number:, recipient_name:, locale:)
+    @phone_number = phone_number
+    @recipient_name = recipient_name
+    @locale = locale
+  end
+
+  def invite
+    url = BASE_URL + "send_invite"
```

Comment:
Lets add some basic metrics from the beginning here so we can create a dashboard to monitor all this.

```
Statsd.increment("imo.invites.attempts")
```

then later on when we get the response

```
Statds.increment("imo.invites.no_imo_account")
# or
Statds.increment("imo.invites.error")
```